assets/app/styles/_extensions.less

```diff
@@ -0,0 +1,9 @@
+.pod-template div[hawtio-extension] {
+  position: relative;
+}
+
+.pod-template .connect-link {
+  position: absolute;
+  right: -5px;
+  top: -66px;
```

Comment:
Absolute positioning to the right would still allow overlapping content, even if we moved container-links to the top. I think we need to leave the link where it is for the moment